node/Dockerfile.s390x

```diff
@@ -12,23 +12,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG QEMU_IMAGE=calico/go-build:latest
-ARG BIRD_IMAGE=calico/bird:latest
+FROM alpine:3.11 as qemu
 
-FROM ${QEMU_IMAGE} as qemu
-FROM ${BIRD_IMAGE} as bird
+ARG QEMU_VERSION=6.1.0-8
```

Comment:
Why switch away from the QEMU_IMAGE and BIRD_IMAGE arguments?

I think we should keep these - this is how all of the other Dockerfiles work and it would be good to keep them consistent.

Reply:
I don't know why but it works when I uses as below:
```
ARG BIRD_VERSION=v0.3.3-188-g0196eee4
ARG ARCH=s390x
ARG BIRD_IMAGE=calico/bird:${BIRD_VERSION}-${ARCH}
```
But it failed with this error if I just use `ARG BIRD_IMAGE`:
```
Dockerfile.s390x:29
--------------------
  27 |
  28 |     FROM calico/bpftool:v5.3-s390x as bpftool
  29 | >>> FROM ${BIRD_IMAGE} as bird
  30 |
  31 |     FROM s390x/alpine:3.8 as base
--------------------
error: failed to solve: failed to parse stage name "calico/bird:-": invalid reference format
make: *** [Makefile:269: .calico_node.created-s390x] Error 1
```

Reply:
I used a default value as this `ARG BIRD_IMAGE=calico/bird:v0.3.3-188-g0196eee4-s390x` to workaround it, so it should be fixed.

Reply:
I don't think that's really a resolution, it just means we're using the defaulted value - we need this to be picked up from the passed in value like it is in the other Dockerfiles.

Reply:
@caseydavenport right, I just removed the default value and it works well. I must messed my local build env so got the error. It's OK we don't set the default value in `ARG BIRD_IMAGE`.